bitwarden.sh

```diff
@@ -60,6 +60,9 @@ echo ""
 function downloadSelf() {
     if curl -L -s -w "http_code %{http_code}" -o $SCRIPT_PATH.1 $BITWARDEN_SCRIPT_URL | grep -q "^http_code 20[0-9]"
```

Comment:
The normal method of controlling permissions to specifically, not to merely copy it us rely on the history of permissions of the previous deployment. Relying on history that is out of the control of the deployment tool is a poor deployment practice.